src/client/containers/formComponent/formComponent.js

```diff
@@ -67,11 +54,11 @@ FormComponent.propTypes = {
   history: PropTypes.instanceOf(Object).isRequired,
   fieldsToRender: PropTypes.instanceOf(Array).isRequired,
   isMember: PropTypes.bool.isRequired,
+  postUserData: PropTypes.func.isRequired,
 };
 
 const mapStateToProps = state => ({
-  postUser: state.postUser.postUser,
-  getUser: state.getUser.getUser,
+  authStatus: state.authStatus,
 });
 
-export default connect(mapStateToProps)(withRouter(FormComponent));
+export default connect(mapStateToProps, { postUserData })(withRouter(FormComponent));
```

Comment:
maybe better to use const mapDispatchToProps = dispatch =>... and pass it as a second arg

Reply:
But if we pass the object of actions as a second parameter then connect function will link our actions with dispatch function and we will not need to create an additional mapStateToProps function. I think this is a redundant code.